modules/api/test/LorisApiDicomsTest.php

```diff
@@ -0,0 +1,362 @@
+        );
+        $this->assertArrayHasKey(
+            'SeriesDescription',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'SeriesDescription',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'SeriesNumber',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'EchoTime',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'RepetitionTime',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'InversionTime',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'SliceThickness',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'Modality',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'SeriesUID',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+    }
+
+    /**
+     * Tests the HTTP POST request for the
+     * endpoint /candidates/{candid}/{visit}/dicoms
+     * // TODO HANDLING OF POST HANDLING NOT IMPLEMENTED
+     *
+     * @return void
+     */
+    public function testPostCandidatesCandidVisitDicoms(): void
+    {
+        $response = $this->client->request(
+            'POST',
+            "candidates/$this->candidTest/$this->visitTest/dicoms",
+            [
+                'headers'     => $this->headers,
+                'http_errors' => false,
+                'json'        => []
+            ]
+        );
+        if ($response->getStatusCode() === 405) {
+            $this->markTestIncomplete(
+                "The method POST is not available for the" .
+                "endpoint candidates/$this->candidTest/$this->visitTest/dicoms"
+            );
+        }
+        // Verify the status code - TODO METHOD NOT ALLOWED YET
+        $this->assertEquals(200, $response->getStatusCode());
+        // Verify the endpoint has a body
+        $body = $response->getBody();
+        $this->assertNotEmpty($body);
+
+    }
+
+    /**
+     * Tests the HTTP GET request for the
+     * endpoint /candidates/{candid}/{visit}/dicoms/tarname
+     *
+     * @return void
+     */
+    public function testGetCandidatesCandidVisitDicomsTarname(): void
+    {
+        $response = $this->client->request(
+            'GET',
+            "candidates/$this->candidTest/$this->visitTest/" .
+            "dicoms/$this->tarfileTest",
+            [
+                'headers'     => $this->headers,
+                'http_errors' => false
+            ]
+        );
+        if ($response->getStatusCode() === 404) {
+            $this->markTestIncomplete(
```

Comment:
The endpoint should exist. No?

Reply:
It exists when the file actually exists. We need to skip to avoid an error, because it is not in the Docker environment. Thus, not skipping causes Travis to fail